mat64/cholesky_example_test.go

```diff
@@ -0,0 +1,58 @@
+package mat64_test
+
+import (
+	"fmt"
+
+	"github.com/gonum/matrix/mat64"
+)
+
+func ExampleCholesky() {
+	a := mat64.NewSymDense(4, []float64{108, -21, -9, -96, -21, 7, 5, 15, -9, 5, 61, 25, -96, 15, 25, 142})
+
+	fmt.Printf("a = %0.4v\n", mat64.Formatted(a, mat64.Prefix("    ")))
+
+	// Compute the cholesky factorization.
+	chol := &mat64.Cholesky{}
```

Comment:
You are using concrete values below (t and test), so do the same here or change those to this form (personally I prefer the concrete form, but in examples I'm not as fussed).